test/api.test.js

```diff
@@ -422,6 +422,138 @@ test('Draw.modes', t => {
   t.end();
 });
 
+test('Draw.combineFeatures -- multiploygon', t => {
+  const [polygonId] = Draw.add(getGeoJSON('polygon'));
+  const [polygon2Id] = Draw.add(getGeoJSON('polygon2'));
+  Draw.changeMode('simple_select', { featureIds: [polygonId, polygon2Id]});
+
+  Draw.combineFeatures();
+  t.equals(Draw.getAll().features.length, 1, 'can combine two features');
+  t.equals(Draw.getAll().features[0].geometry.type, 'MultiPolygon', 'can merge two polygons into MultiPolygon');
+  Draw.deleteAll();
```

Comment:
Lets add a test like

`t.deepEquals(Draw.getAll().features[0].geometry.coordinates[0], geoGeoJSON('polygon').geometry.coordinates)`

So we know that the combine function is working as expected.